### PR TITLE
use only github.com/google/uuid for uuid generation

### PR DESCRIPTION
**Pull request by mmorel-35**
**What this PR does / why we need it**:

This use only one library as direct dependency for uuid generation as it was done on go-kong and deck before

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [x] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: internal/mgrutils/reports.go [hunk 1]
```diff
@@ -28,10 +28,7 @@ func RunReport(ctx context.Context, kubeCfg *rest.Config, kongCfg sendconfig.Kon
 	}
 
 	// create a universal unique identifier for this system
-	uuid, err := uuid.GenerateUUID()
-	if err != nil {
-		return fmt.Errorf("failed to generate a random uuid: %w", err)
-	}
+	uuid := uuid.NewString()
```
Comment (mflendrich) [inline internal/mgrutils/reports.go:31]: As required by the unchecked checkbox in the PR description: please add a changelog entry to CHANGELOG.md under the "TBD" release that we're changing the uuid generation mechanism.

**Review by mflendrich — COMMENTED** [review]
One chore left, otherwise LGTM

**Review by mflendrich — APPROVED** [review]

**Review by mflendrich — APPROVED** [review]

**Review by shaneutt — APPROVED** [review]

**Comment by mmorel-35** [comment]
About the licenses job failing for not members of Kong. Have you seen https://github.com/github/licensed ? There are even actions for automation : https://github.com/github/licensed#github-actions

**Comment by shaneutt** [comment]
> About the licenses job failing for not members of Kong. Have you seen https://github.com/github/licensed ? There are even actions for automation : https://github.com/github/licensed#github-actions

Yeah we need some improvements here, I haven't personally looked into this alternative: please do feel free to file an issue we can take a look at an alternative CI setup for licensing :+1:

**Pull request merged**